src/components/Admin/Preview/ImagePreview.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './ImagePreview.css';
+
+const ImagePreview = props => {
+  return (
+    <div className="preview-image">
+      <img src={props.image.uri} alt={props.image.id} />
+      <br />
```

Comment:
Instead of using `<br />` which is mostly a purely visual element, we can do this with block level elements (or css if we wanted custom styles) - we're doing this in a few places

e.g.

```
<img src={...}/>
<div><button></div>
```